frontend/packages/topology/src/actions/provider.ts

```diff
@@ -58,3 +59,12 @@ export const useTopologyVisualConnectorActionProvider = (element: Edge) => {
     return [actions, !inFlight, undefined];
   }, [actions, inFlight]);
 };
+
+export const useVmTopologyActionsProvider = (element) => {
+  const vm: VMKind = getResource(element);
```

Comment:
Again this introduces a dependency on kubevirt package. Shouldn't this action be a generic action for any resource?